### ISSUE TITLE
Make sure TUD accounts are reset to back to their original state on checkin

### ISSUE THREAD

**Issue by holdenhinkle** [department-of-veterans-affairs/va.gov-team#34361]
Make sure TUD accounts are reset to back to their original state on checkin.

**Comment by holdenhinkle** [comment]
Draft PR - https://github.com/department-of-veterans-affairs/vets-api/pull/8729

^ is untested. I can't login to `vets-website` locally with `id.me` accounts for some reason.

**Comment by holdenhinkle** [comment]
The code for this is in [this branch](https://github.com/department-of-veterans-affairs/vets-api/pull/8721) for now. I made changes to the `tud_accounts` table in this branch which prevents TUD from running locally using the original branch for this update and I don't want to rollback the migration.

**Comment by pjhill** [comment]
This can be merged once GitHub OAuth can be merged.

**Comment by holdenhinkle** [comment]
This has been finished over a month. Merging is blocked on this until Oauth is working. We just need to fix the redirects issue.

**Comment by holdenhinkle** [comment]
This was merged on Friday, with the PR that introduced TUD support for login.gov accounts.

**Comment by pjhill** [comment]
Yay for clearing this now that the GH OAuth issue and Login.gov support issue have been resolved. This feature could probably use with a concerted effort to test now that it is merged. We can accomplish that in a future sprint under a separate task.
